backend/scripts/update_licenses.py

```diff
@@ -11,7 +12,7 @@
 EXCEPTIONS_URL = 'https://raw.githubusercontent.com/spdx/license-list-data/v{}/json/exceptions.json'
 
 
-def download_data(url):
+def download_data(url: Union[httpx.URL, str]):
```

Comment:
you could also add here `from __future__ import annotations` and use new syntax

Reply:
Ah, I haven't looked at the scripts directory yet only hatchling and hatch, I'll have a look through it now.